src/java/org/apache/cassandra/io/sstable/metadata/MetadataCollector.java

```diff
@@ -103,7 +100,10 @@ public static StatsMetadata defaultStatsMetadata()
     protected boolean hasLegacyCounterShards = false;
     protected long totalColumnsSet;
     protected long totalRows;
+    protected MinMaxLongTracker keyRangeTracker = new MinMaxLongTracker(Long.MAX_VALUE, Long.MIN_VALUE);
```

Comment:
The constructor is (defaultMin, defaultMax). Any reason you're using long.max for defaultMin instead of the other way? Won't affect anything as long as the tracker is updated at least once before being queried for min/max

Reply:
As a default state, which could easily be checked for if needed.